inst/include/Rcpp/DataFrame.h

```diff
@@ -83,6 +83,26 @@ namespace Rcpp{
             return LENGTH(rn);
         }
 
+        // Leave in place for a year
```

Comment:
The goal behind the PR was to align the `DataFrame` class to match with:

1. the _R_ functions `ncol()` and `nrow()`;
2. the `Matrix` class member functions `.nrow()`, `.ncol()`, `.rows()`, `.cols()`.

In an ideal world, the old interface _probably_ should be cleaned up as a result. But, it most certainly can be left in place and the `nrows()` aspect can be _omitted_ from documentation.

By my count, the following ~20 packages on CRAN would be affected:

- `BacArena`
- `Biocomb`
-  `Cyclops`
- `deepboost`
- `dplyr`
- `FeatureHashing`
- `FIT`
- `feather`
- `gaston`
-  `medfate`
- `missDeaths`
- `Rfast`
- `Rborist`
- `raptr`
- `readr`
- `readstat13`
- `reshape2`
- `tweenr`
- `valr`
- `wsrf`